resources/assert_lib.brs

```diff
@@ -65,6 +66,19 @@ sub __isFalse(actual, error)
     end if
 end sub
 
+sub __isValid(actual, error)
+    if actual <> invalid then
+        m.__pass()
+    else
+        m.__fail(m.formatError({
+            message: error,
+            actual: actual,
+            expected: "non-invalid",
```

Comment:
This might be a little strange, but this seems like the best way to represent the expected value 🤷

Reply:
Yeah... I've been thinking A LOT about what should I pass there. Definitely more than I had to. 😄  But I'm actually satisfied with this one.